synapse/config/server.py

```diff
@@ -679,6 +679,17 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
             config.get("exclude_rooms_from_sync") or []
         )
 
+        delete_stale_devices_after: Optional[str] = (
+            config.get("delete_stale_devices_after") or None
+        )
```

Comment:
Why the `or None`? `get(...)` by default will return `None` if it doesn't exist. Is this trying to handle bad values like `False` or `0` or something?

Reply:
Yeah basically that. It's mostly a habit I took up after getting a few comments about it in reviews.

Reply:
Alright, I guess I find it a bit odd with the `None`, but seems OK. 👍